modules/microshift-manifests-overview.adoc

```diff
@@ -6,12 +6,7 @@
 [id="microshift-manifests-overview_{context}"]
 = How manifests work with kustomize
 
-Providing multiple directories allows a flexible method of managing {product-title} workloads. When you run the `kustomize` configuration management tool, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the manifests to the cluster.
-
-[IMPORTANT]
-====
-The `kustomize` tool must be either running as a separate step in the boot process, or it must be part of the {product-title} image.
-====
+The `kustomize` configuration management tool is integrated with {product-title}. At every start, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the `kubectl apply -k` command to apply the identified manifests to the cluster.
```

Comment:
```suggestion
The `kustomize` configuration management tool is integrated with {product-title}. At every start, {product-title} searches the `/etc/microshift/manifests` and `/usr/lib/microshift/` manifest directories for a `kustomization.yaml` file. If it finds one, {product-title} automatically runs the equivalent of the `kubectl apply -k` command to apply the identified manifests to the cluster.
```

It doesn't actually run the "kubectl" command. The same code is built into microshift, but it runs differently.